hw/ip/alert_handler/dv/env/alert_handler_env.core

```diff
@@ -9,6 +9,7 @@ filesets:
     depend:
       - lowrisc:dv:cip_lib
       - lowrisc:dv:gen_ral_pkg
+      - lowrisc:ip:alert_handler_component
```

Comment:
This is also added to alert_handler_sim.core

I would like to understand more why these errors are suddenly showing up?

Reply:
@cindychip 
Is the alert_handler_env using anything at all from the design? Ideally, DV code should not be referencing anything from the RTL area. If there is a params pkg that could theoretically be shared with DV, that it should be a separate standalone core file.

Reply:
well yes, we do use the `alert_pkg.sv` file's : `alert_pkg::NAlerts` and `alert_pkg::N_ESC_SEV` to get number of alerts and number of escalators. I think it would be good separate this `alert_pkg.sv` to a standalone core file.

Reply:
I see that the i2c testbench relies on the generated register package. do we have other testbenches that have such a dependency? In that case it might make sense to standardize on this and mandate each comportable IP has a pkg core file and a "main" core file...